### ISSUE TITLE
Making --minimum-mapping-quality to haplotypecaller a parameter?

### ISSUE THREAD

**Issue by ksahlin** [bioinf-benchmarking/mapping-benchmarking#2]
Would it be easy to make --minimum-mapping-quality to haplotypecaller a parameter in the pipeline?

One thing that we (@marcelm and I) are interested in is to better set the MAPQ score in strobealign. I am thinking whether the magic value of 20 (as recommended for haplotypecaller) might have been be tuned for BWA or so. Would be good o have this as a parameter for feedback.

**Comment by ivargr** [comment]
Yes, that should be fairly easy to implement in the current pipeline, and sounds like an interesting thing to support. I'll have a look at it within the next few days:)

**Comment by ksahlin** [comment]
I saw that you updated the benchmark, nice! 

Some questions:

1. The "Variant calling accuracy vs mapq threshold"-plot: the x-axis look a bit strange, perhaps you plotted precision instead of 1-precision?

2. Is the "Accuracy" in the "Accuracy vs read length vs error rate" referring to the mapping accuracy? (i.e. how many reads are aligned to the correct genomic location? 

If the answer to 2 is yes, then I guess we can take from your analysis is that us working on strobealign (currently @marcelm and I) should look into more accurately setting the MAPQ values, as the mapping accuracy, which looks high, is not reflected in the variant calling results (with haplotypecaller).

**Comment by ivargr** [comment]
Yes, I uploaded the plots, but I was a bit quick doing so, so I might have introduced some bugs in some of the metrics/simulations. 

1) The 1-precision is precision as you point ut, will fix. I'll also add mapq labels to the dots so that it makes more sense.
2) The accuracy in "Accuracy vs. read length and error profile" is F1-score (computed from precision and recall, computed by hap.py.

There is something a bit funky with the read simulation which I'm currently debugging, so I wouldn't trust these results 100% yet. I'm currently debugging a bit and writing some tests, I'll let you know when you can start trusting the results.

From what I've seen so far, Strobealign seems to be worse at assigning mapq scores than some of the other aligners (without necessarily having worse overall accuracy), so there might be a lot of potential in improving the mapq calculation. I'll try to generate some plots that better show this.
